www/blog/2022-11-06-announcing-trpc-10.mdx

```diff
@@ -0,0 +1,89 @@
+author_title: Creator of tRPC
+author_url: https://twitter.com/alexdotjs
+author_image_url: https://avatars1.githubusercontent.com/u/459267?s=460&v=4
+---
+
+import { GithubSponsorButton } from '@site/src/components/GithubSponsorButton';
+
+## Introduction
+
+tRPC's mission from day one has been to provide you with an easy way of moving fast and not breaking nothing whilst building full-stack applications.
+
+Since we released our last major version in August 2021, we've gotten over 13,500+ more stars on GitHub, a [Discord-community](https://trpc.io/discord) with over 2,000 members, thousands more applications built using tRPC, a growing ecosystem, and countless community members contributing and providing valuable feedback.
+
+Today, **we're launching tRPC 10**. It is already used in production and can safely be used for in your next, or current, TypeScript project.
+
+Get up and running with the [example applications](https://trpc.io/docs/v10/awesome-trpc#-starting-points-example-projects-etc) to get a feel of tRPC 10, or see the [guide for migrating from tRPC 9 to tRPC 10](https://trpc.io/docs/v10/migrate-from-v9-to-v10).
+
+## Overview of changes
+
+tRPC 10 is our biggest release ever. This is the first time we have made any fundamental changes to the structure of tRPC, yet these changes unlock new possibilities for cross-functional teams working on cutting-edge applications.
+
+tRPC 10 includes:
+
+- **Massively Improved TypeScript Performance:**
+- **Improved API Experience**
+  - Calling functions instead of invoking strings
+  - CMD+clicking and refactoring from the frontend
+  - Easier to infer types
+- **Powerful Backend Framework**
+  - Reusable middlewares
+  - Reusable base procedures
+
+### Massively Improved TypeScript Performance
+
+TypeScript enables us to do very cool things, but it comes at a cost. Many of the things that we (ab)use in tRPC are excessive on the compiler, and as a result, you might have experienced your IDE slowing down as your application grows.
+
+In v10, we've improved this massively by changing our API allows TypeScript to be much faster, especially during incremental compilation, allowing your IDE to remain snappy.
+
+### Improved DX
+
+tRPC 10 embraces TypeScript to its fullest potential to give you an unparalleled DX across the API.
+
+You will notice the following in tRPC 10:
+
+- Jump straight to your backend function from your frontend by just CMD+clicking the path
+- Right click and _"Rename Symbol"_ to rename an input argument or a procedure name across your whole application.
+- [Easier to infer types](https://trpc.io/docs/v10/infer-types)
+
+### Powerful Backend Framework
+
+We've doubled down on the core philosophy of making cross-functional teams more effective and enabling more developers to jump into the backend.
+
+tRPC 10 creates an seamless experience between the front- and backend that further blurs the line between them
+
+Other ways tRPC 10 provides a powerful backend framework:
```

Comment:
```suggestion
tRPC 10 provides an even more powerful backend framework than before. Building your API has never been easier:
```
the seamless experience between front and backend line does not fit in a section about tRPC's backend capabilities